test/ctia/bulk/core_test.clj

```diff
@@ -16,3 +23,24 @@
                                  {:ConfigService {:get-in-config get-in-config}
                                   :StoreService {:get-store (constantly nil)}})]
       (is (= [nil] res)))))
+
+(defn schema->keys
+  "Extracts both: required and optional keys of schema as set of keywords"
+  [schema]
+  (let [reqs (->> schema st/required-keys keys)
+        opts (->> schema st/optional-keys keys (map :k))]
+    (set (concat reqs opts))))
+
+(deftest bulk-schema-excludes-disabled-test
+  (testing "ensure NewBulk schema includes only enabled entities"
+    (s/set-fn-validation! false) ;; otherwise it fails for incomplete :APIHandlerServices (we only need :FeaturesService for the test)
+    (with-app-with-config app
+      [features-svc/features-service] {:ctia {:features {}}}
+      (let [bulk-schema (NewBulk (app/service-graph app))]
```

Comment:
This could be cleaner if you write a utility function that selects part of the service graph like https://github.com/threatgrid/ctia/blob/e6b45249024c73093ae7692e9b7ff1cbef207f7d/test/ctia/test_helpers/http.clj#L45-L50

Once you decide a name for your schema in the [other conversation](https://github.com/threatgrid/ctia/pull/1061/files#r556857544), just write a function `app->MyName` as above and use it to avoid "passing the world" here.

Using this style results in a stronger test, as we can be certain the test passes with minimal access to the app state.

Reply:
@frenchy64 Can you give it another gander: https://github.com/threatgrid/ctia/pull/1061/commits/ede787938887ce2bafe8bf909aa649a5c78f413f
thanks!